libp2p/network/swarm.py

```diff
@@ -128,11 +126,10 @@ def set_stream_handler(
         try:
             secured_conn = await self.upgrader.upgrade_security(raw_conn, peer_id, True)
         except SecurityUpgradeFailure as error:
-            # TODO: Add logging to indicate the failure
+            error_msg = "fail to upgrade security for peer %s" % peer_id
+            logger.debug(error_msg)
```

Comment:
Would it be more efficient to use `logger.debug("fail to upgrade security for peer %s", peer_id)`, for the log aggregation purpose?